modules/jaeger-removing-instance-cli.adoc

```diff
@@ -34,28 +43,38 @@ tracing2                 1/1     1            1           35m
 +
 . To remove an instance of Jaeger, run the command:
 +
+[source,terminal]
 ----
-oc delete jaeger <deployment-name>  -n <jaeger-project>
+oc delete jaeger <deployment-name> -n <jaeger-project>
 ----
 +
 For example,
 +
+[source,terminal]
 ----
 oc delete jaeger tracing2 -n jaeger-system
 ----
 +
 
-. To verify the deletion, run `oc get deployment` again :
+. To verify the deletion, run `oc get deployment` again:
 +
+[source,terminal]
 ----
 oc get deployments -n <jaeger-project>
 ----
 
 +
 For example,
 +
+[source,terminal]
 ----
 oc get deployments -n jaeger-system
```

Comment:
Does this need a `$`?

Reply:
OMG, I must have been completely out of $$$ that day.  Fixed.